### PR TITLE
cx_Freeze 6.6

### PR DESCRIPTION
**Pull request by marcelotduarte**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
   url: https://pypi.io/packages/source/c/cx_Freeze/cx_Freeze-{{ version }}.tar.gz
-  sha256: e0d03cabcdf9b9c21354807ed9f06fa9481a8fd5a0838968a830f01a70820ff1
+  sha256: c4af8ad3f7e7d71e291c1dec5d0fb26bbe92df834b098ed35434c901fbd6762f
 
 build:
   skip: true  # [osx and py==39]
   number: 0
   script:
-    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+    - {{ PYTHON }} -m pip install cx_Logging --no-deps --ignore-installed --no-cache-dir -vv -i https://pypi.org/simple  # [win]
```
Comment (marcelotduarte) [inline recipe/meta.yaml:15]: @jschueller I need the dependent cx_logging. As it is not in conda and tried "pip install cx_Logging". But fail.
How I can bypass "Ignoring indexes: https://pypi.org/simple" by pip. It has an environment variable to change or what?

Comment (jschueller) [reply recipe/meta.yaml:15]: Maybe it's best to create a package for it

Comment (jschueller) [reply recipe/meta.yaml:15]: maybe it's one of these:
set "PIP_NO_BUILD_ISOLATION=False" 
set "PIP_NO_DEPENDENCIES=True" 
set "PIP_IGNORE_INSTALLED=True" 
set "PIP_CACHE_DIR=D:\bld\cx_freeze_1618505399456\pip_cache" 
set "PIP_NO_INDEX=True"

but maybe its best to create a dedicated package ?

Comment (marcelotduarte) [reply recipe/meta.yaml:15]: > but maybe its best to create a dedicated package ?

Yes, it is. Can you create?

Comment (jschueller) [reply recipe/meta.yaml:15]: I'm under the water here, sorry

Comment (marcelotduarte) [reply recipe/meta.yaml:15]: Is this slang?

**Comment by marcelotduarte** [comment]
@conda-forge-admin, please rerender

**Comment by marcelotduarte** [comment]
@jschueller can you merge this PR?

**Pull request merged**